langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/LangLibArrayTest.java

```diff
@@ -333,6 +333,24 @@ public void testTupleSetLengthIllegal() {
         Assert.fail();
     }
 
+    @Test(expectedExceptions = BLangRuntimeException.class,
+            expectedExceptionsMessageRegExp =
+                    "error: \\{ballerina/lang.array}IndexOutOfRange \\{\"message\":\"array index out of range: " +
+                            "index: 2, size: 1\"}.*")
+    public void testForEachWhileRemoveElements() {
+        BRunUtil.invoke(compileResult, "testForEachWhileRemoveElements");
+        Assert.fail();
```

Comment:
Do we need `Assert.fail();` in both tests? Can't we omit them?

Reply:
Removed unnecessary uses of `Assert.fail()`